internal/commands/storage/import.go

```diff
@@ -69,6 +70,7 @@ func (s *importCommand) InitCommand() {
 	flagSet.StringVar(&s.sourceLocation, "source-location", "", "Location of the source of the import. Can be a file or a URL.")
 	flagSet.StringVar(&s.existingStorageUUIDOrName, "storage", "", "Import to an existing storage. Storage must be large enough and must be undetached or the server where the storage is attached must be in shutdown state.")
 	config.AddToggleFlag(flagSet, &s.noWait, "no-wait", false, "Do not wait until the import finishes. Only applicable when importing from a remote URL.")
+	config.AddToggleFlag(flagSet, &s.wait, "wait", false, "Wait for storage to be in online state before returning.")
```

Comment:
If both `--no-wait` and `--wait` are used, `--no-wait` takes precedence as that code branch does not check for `--wait` value. Didn't add explanation to help texts as I assumed that these are not likely used together 😅

Reply:
This is bit confusing. My first thought would be that If `--no-wait` is not set then we should wait.

Reply:
Yeah, currently `--no-wait` returns after url-to-cloud import is started and does not affect other import types. So it is has different scope than this new `--wait` would have.

Renaming the `--wait` parameter to `--wait-online` would make this bit clearer without being a breaking change. However, then we would have `--wait` parameter for async-by-default server operations and `--wait-online` paramenter for async-by-default storage operations.

Not sure what would be the best approach here.

Reply:
Yeah I guess we can't have perfect solution without doing breaking change but I would vote for changing it to `--wait-online`  and then sort these naming conflicts out in v3.

Reply:
I would vote for keeping the new flag as `--wait` and somehow explain in the help how these two are (not) related. Leaning towards this to as the server's `--wait` more or less already acts in a similar fashion from the user point of view.

In that case, we could rename `--no-wait` to something more import-related in the next major release.

Reply:
I tried to modify `--no-wait` description to make the difference more clear, and created a ticket for renaming it in next major release.